### ISSUE TITLE
Convert release note linebreaks to HTML breaks

### ISSUE THREAD

**Issue by lordscarlet** [nathanwoulfe/Plumber-2#64]
**Describe the solution you'd like**
It is hard to parse the release notes when they run together in a single line

![image](https://user-images.githubusercontent.com/17951/104646569-99bb7780-567e-11eb-826f-2d8946172c48.png)

**Comment by lordscarlet** [comment]
I also need to figure out why our is reporting the wrong version.. 
![image](https://user-images.githubusercontent.com/17951/104647012-3f6ee680-567f-11eb-9f92-82ba42101d59.png)

**Comment by nathanwoulfe** [comment]
Hi @lordscarlet, I'm guessing you're running a nightly? Latest release is 1.4.3, latest nightly will be 1.4.4.x, which will then mean the check for the release notes will return dodgy info, since it just does a simple compare between two values. Really should be checking that the installed version is lower than the latest (as reported from the github api).

Should be easy enough to get linebreaks into those notes - it all comes down the wire from GH, so should be able to request it as HTML.

**Comment by nathanwoulfe** [comment]
Was missing a header in the GB request, looks much better now!

![image](https://user-images.githubusercontent.com/3248070/104669632-279e6d80-5726-11eb-98aa-9784fb86672e.png)

**Comment by lordscarlet** [comment]
Lovely!

**Comment by nathanwoulfe** [comment]
Included in 1.5.0
